src/tags_int.cpp

```diff
@@ -2833,6 +2860,18 @@ namespace Exiv2 {
         return EXV_PRINT_TAG(exifNormalSoftHard)(os, value, metadata);
     }
 
+    extern const TagDetails exifCompositeImage[] = {
+        { 0, N_("Unknown") },
+        { 1, N_("NonComposite")   },
+        { 2, N_("GeneralComposite")   },
+        { 3, N_("CompositeCapturedWhenShooting")   }
+    };
+
+    std::ostream& printCompositeImage(std::ostream& os, const Value& value, const ExifData* metadata)
```

Comment:
Let's use the macro EXV_PRINT_TAG() instead, no need to clutter the API with functions that are almost never used...

Reply:
I agree.  The code was cut'n'pasted without really thinking/understanding how everything fits together.  We shouldn't add function to the API without good reason.